tests/task/task_runner/test_standard_task_runner.py

```diff
@@ -121,11 +121,16 @@ def test_on_kill(self):
         runner = StandardTaskRunner(job1)
         runner.start()
 
-        # give the task some time to startup
-        time.sleep(3)
-
+        # Give the task some time to startup
+        time.sleep(10)
```

Comment:
Why so much more time than before?

Reply:
This test is flaky, but now with the loop below I think we can go back to 3s.

Reply:
The problem is with wrong dependencies epoch number change. APT-deps are re-installed from the scratch in this PR which makes it take longer. I will explain it in comment to the PR.